pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -545,9 +545,11 @@ type ClientSpec struct {
 }
 
 type CleanupPolicySpec struct {
-	DeleteDataDirOnHosts string `json:"deleteDataDirOnHosts"`
+	Confirmation CleanupConfirmationProperty `json:"confirmation,omitempty"`
```

Comment:
Can we just use a string here? IMO the type definitions of strings make things more complicated.
```suggestion
	Confirmation string `json:"confirmation,omitempty"`
```

Reply:
Having a type allows us to do `cluster.Spec.CleanupPolicy.Confirmation = cephv1.DeleteDataDirOnHostsConfirmation` which is nice IMHO.

Reply:
If they are both strings it seems the same for readability.

Reply:
To be discussed another day...